components/hubspot/actions/search-crm/search-crm.mjs

```
@@ -201,6 +209,23 @@ export default {
         label: labels.plural,
       })) || [];
     },
+    async paginate(params) {
+      let results;
+      const items = [];
+      while (!results || params.after) {
+        results = await this.hubspot.searchCRM(params);
+        if (results.paging) {
+          params.after = results.paging.next.after;
+        } else {
+          delete params.after;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)